data/mysql/schema.sql

```diff
@@ -625,7 +625,8 @@ CREATE TABLE `ezcontentobject` (
   KEY `ezcontentobject_lmask` (`language_mask`),
   KEY `ezcontentobject_owner` (`owner_id`),
   KEY `ezcontentobject_pub` (`published`),
-  KEY `ezcontentobject_status` (`status`)
```

Comment:
Please apply these changes also to `/eZ/Publish/Core/Persistence/Legacy/Tests/_fixtures/schema.**.sql` scripts